frontend/src/lib/components/LemonModal/LemonModal.scss

```diff
@@ -33,6 +36,7 @@
     margin: 1rem auto;
     border-radius: var(--radius);
     background-color: #fff;
+    border: 1px solid var(--border);
```

Comment:
This is in the design but was missed out. Noticeably better if the blur isn't supported